plugins/rts54hub/fu-rts54hub-rtd21xx-device.c

```diff
@@ -18,26 +18,46 @@ struct _FuRts54hubRtd21xxDevice
 	guint8			 target_addr;
 	guint8			 i2c_speed;
 	guint8			 register_addr_len;
+	guint8			 fwupdate_mode;
```

Comment:
I suggest making this boolean.  For example `gboolean background_update`

Reply:
The "guint8"  has better expansibility. If there is another burning method, it can also be supported.

Reply:
I think it should be `FuRt21xxFwUpdaMode` then, rather than guint8

Reply:
yes , FuRt21xxFwUpdaMode  is better. 
We'll  use FuRt21xxFwUpdaMode instead. thanks.